samples/matter/light-switch/readme.rst

```diff
@@ -0,0 +1,170 @@
+
+- Blinking slowly indicate that Matter BLE Advertisment is active for Matter Provisioning
+- Fast Blink (100ms) Means that BLE connection is active and Matter Provisioning is ongoing
+
+Red LED:
+
+- Fast Blinking (100ms) indicate Thread network is active for joining provisionmed newtwork
+- 1 Second constant Active Mode indicate That Matter controller subscribed is establis,ent or resumed
+- Single short active indicate Endpoint activation
+
+Sample support Balletto Devkit Joystick buttons
+
+Center Button ``SW0``:
+
+- Press down will toggle OnOff-switch endpoint state
+
+Left Up Corner Button ``SW1``:
+
+- Generic switch for generate ``InitialPress`` at active state  and  ``ShortRelease`` after release button.
+
+Right Up Corner Button ``SW2``:
+
+- Hold over 3 seconds for doing Matter factoryreset. After Factory reset press DK normal Reset Button for activate BLE advertisment.
+
+Provisioning with Apple Home
+****************************
+
+1. Flash builded binary to device and press ``Reset`` -button.
+#. After reset you shuold see slowly blinking blue led which indicate that Device is advertisment Matter ready device for provision.
+#. Open ``Apple Home`` application with Iphone or Ipad which is connected to same Wifi with Matter Conttroler and Thread border router.
+#. Click ``+`` -button from app for ``Add Accessory``.
+#. Scan QR code
+   
+   .. image:: QRCode.png
+
+#. Click ``Add To Home`` for Start provisioning to Thread neywork and Matter fabric.
+#. You shuold see now fast blinking Blue led when BLE connection is established.
+#. ``Aplle Home`` will warning Uncertificated Acessory so just click ``Add Anyway`` for continue.
+#.  When Provision is ready ``Apple Home`` ask where to add light select a room and click ``Continue``.
+#. Now you can rename a light default is ``Matter Accessory`` name for example ``Matter Switch`` and click ``Continue``.
+#. Click ``Continue``
+#. Click at next window ``View at Home`` and you can see light switch .
+
+Device is now provisioned and will report automatically if user Press ``SW0`` On/Off Light-switch  or ``SW1`` Generic switch.
+Device is now provisioned and can control provisioned Matter Light(s) by adding ``Automation`` at ``Apple Home`` for connected matter device light switch.
+
+Light-switch automation with Apple Home
+***************************************
+
+Light on and off need separate ``Automation``. Next flow example is for Matter Accessory which name is ``Matter Light``
+
+How to create Light On ``Automation``:
+
+1. Click ``+``-button from app for ``Add Automation``.
+#. Select ``An Accessory is Controlled`` for use ``Matter Switch`` controlling a ``Matter Light``.
+#. Select ``Matter Switch`` from list or your own named light switch device and click ``Next``.
+#. Next page select ``Turn On`` and click ``Next``.
+#. Select from list ``Matter Light`` and click ``Next``.
+#. Click ``Matter Light`` light icon for setting ``Turn on`` light dimm level and click ``Done``.
```

Comment:
`Turn on`` light dimm level and click ``Done`` ->
`Turn on`` select dimming level and click ``Done``